docs/src/pages/demos/chips/Chips.js

```diff
@@ -30,6 +30,7 @@ function Chips(props) {
   return (
     <div className={classes.root}>
       <Chip label="Basic Chip" className={classes.chip} />
+      <Chip label="Clickable without onClick Chip" clickable className={classes.chip} />
```

Comment:
@mbrookes Unless I'm missing something, by demo, the point was to showcase a **link chip** no?

Reply:
😊

Reply:
Yes. For buttons we link to the page section id, so a link to '#chip' would do.

Reply:
Got it. you also want me to add component 'a' ? or just adding a 'href="#chip"' will do? And what title will be suitable for it? "Link" or "Clickable without onClick chip (Link)".

Reply:
Whatever use case you're solving for, and as short a name as possible.

Reply:
@mbrookes I have added link to the Chip and also made the name as Clickable Link Chip. Please let me know if I have to make any more changes